lib/helpers.js

```diff
@@ -21,5 +23,24 @@ var getLogger = function getLogger(optLogLevel, optTransport) {
   return bind(logger.log, logger);
 };
 
+// TODO we can do this better
+var versionStr = module.exports.repository.replace('/', ':') + '/' + module.exports.version + ' '
+  + os.platform() + '/' + os.release() + ' '
+  + 'node/' + process.version.replace('v', '');
+
+var getVersionString = function getVersionString() {
+  return versionStr;
+};
+
+var appendToVersionString = function appendToVersionString(name, version) {
```

Comment:
can we put some JSDoc on this function so that its surfaced by our reference docs? the intention is for this to be used by "higher-level" users of the SDK, right?